.github/workflows/lab-quic-callee.yml

```diff
@@ -31,65 +32,18 @@ jobs:
             Write-Host "Failed to alert observer VM online: $_"
             exit 1
           }
-  
-  #
-  # Build Jobs
-  #
-
-  build-windows:
-    name: Build WinUser
-    needs: [netperf-register]
-    strategy:
-      fail-fast: false
-      matrix:
-        os: ['windows-2022']
-        tls: [schannel] # , openssl, openssl3]
-    uses: microsoft/msquic/.github/workflows/build-reuse-win.yml@main
-    with:
-      os: ${{ matrix.os }}
-      tls: ${{ matrix.tls }}
-      build: -Perf
-      ref: ${{ github.event.client_payload.sha || github.event.client_payload.ref || inputs.ref || 'main' }}
-
-  build-windows-kernel:
-    name: Build WinKernel
-    needs: []
-    strategy:
-      fail-fast: false
-      matrix:
-        os: ['windows-2022']
-    uses: microsoft/msquic/.github/workflows/build-reuse-winkernel.yml@main
-    with:
-      os: ${{ matrix.os }}
-      ref: ${{ github.event.client_payload.sha || github.event.client_payload.ref || inputs.ref || 'main' }}
-
-  build-unix:
-    name: Build Unix
-    needs: []
-    strategy:
-      fail-fast: false
-      matrix:
-        include:
-          - os: 'ubuntu-20.04'
-            tls: 'openssl'
-          - os: 'ubuntu-22.04'
-            tls: 'openssl3'
-    uses: microsoft/msquic/.github/workflows/build-reuse-unix.yml@main
-    with:
-      os: ${{ matrix.os }}
-      tls: ${{ matrix.tls }}
-      xdp: ${{ matrix.xdp }}
-      build: -Perf
-      ref: ${{ github.event.client_payload.sha || github.event.client_payload.ref || inputs.ref || 'main' }}
-  
 
   run-secnetperf:
     name: secnetperf
-    needs: [build-windows, build-windows-kernel, build-unix]
+    needs: [netperf-register]
     runs-on:
     - self-hosted
     - ${{ github.event.client_payload.assigned_runner }}
     steps:
+    - name: Checkout code
```

Comment:
It's slightly surprising to see a "checkout code" step added in a change that is moving the build out of the callee- what's this for?

Reply:
Also, the vague "checkout code" clashes with the existing step "checkout microsoft/msquic".

Reply:
"Checkout code" means to check out this repository (netperf), so that I could have access to ./netperfrepo/actions/download-caller-artifacts.

"Checkout microsoft/msquic" grabs various scripts relevant to the project netperf is being used for.

Reply:
I see! That naming does seem idiomatic then. LGTM.